scanner/metadata/metadata.go

```diff
@@ -148,6 +149,14 @@ func (t Tags) ModificationTime() time.Time { return t.fileInfo.ModTime() }
 func (t Tags) Size() int64                 { return t.fileInfo.Size() }
 func (t Tags) FilePath() string            { return t.filePath }
 func (t Tags) Suffix() string              { return strings.ToLower(strings.TrimPrefix(path.Ext(t.filePath), ".")) }
+func (t Tags) BirthTime() time.Time {
+	times, _ := times.Stat(t.filePath)
```

Comment:
The variable should be name something else, as `times` collides with the package name.
Also, you must check for errors here, as the Stat function can return `null` if there is an error.